docroot/themes/humsci/humsci_basic/src/scss/ckeditor/imports.scss

```diff
@@ -322,6 +322,19 @@ $px-only: true;
       }
     }
   }
+
+  // Fix Video preview in CKeditor for "Align center and break text" option
+  .drupal-media-style-align-center {
+    max-width: none;
+
+    [data-media-source='video'] {
+      max-width: 100%;
+    }
+
+    [data-media-source='image'] {
+      max-width: 25%;
```

Comment:
Please use `50%` here, to ensure that the image looks exactly the same as it does in the current version.

Also, keep in mind the following:
- There are more media types besides images and video. All the other media types should look exactly the same as before adding these changes. Maybe you can use `not()` to target everything except video.
- The media element should be centered. Maybe you need to add `margin-left: auto; margin-right: auto;`

Reply:
Thank you!!!